src/main/kotlin/baseball/view/OutputView.kt

```diff
@@ -0,0 +1,26 @@
+package baseball.view
+
+import baseball.model.Score
+import baseball.util.GameMessage
+import java.lang.StringBuilder
+
+class OutputView {
+
+    fun printGameStartMessage() = println(GameMessage.GAME_START_MESSAGE)
+    fun printGameInputMessage() = print(GameMessage.GAME_INPUT_NUMBER_MESSAGE)
+    fun printGameSuccessMessage() = println(GameMessage.GAME_SUCCESS_MESSAGE+" "+GameMessage.GAME_OVER_MESSAGE)
```

Comment:
```suggestion
    fun printGameSuccessMessage() = println("${GameMessage.GAME_SUCCESS_MESSAGE} ${GameMessage.GAME_OVER_MESSAGE})
```

코틀린에서 이렇게 쓸 수 있습니다. 
String 끼리 더하면 메모리를 더 잡기 때문에 이렇게 쓰는게 좋아요.

Reply:
아 더하는게 메모리를 더 사용하는줄 몰랐는데 감사합니다!